bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,65 @@
+import random
+
+from cowsay import char_names, get_output_string
+from discord import Embed
+from discord.ext import commands
+
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+
+class Cowsay(commands.Cog):
+    """Cog for the cowsay command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(
+        aliases=["cow"],
+        help=f"""
+        Generates a cowsay string and sends it. The available characters for cowsay are:
+
+        {', '.join(char_names)}"""
+    )
+    async def cowsay(self, ctx: commands.Context, character: str = "Cow", *, text: str = None) -> None:
+        """
+        Generates some cowsay ASCII art and sends it in Discord.
+
+        There are multiple character options to choose from. Here is a list:
+        """
+        if not text:
+            text = f"I'm a {character.lower()}"
+        text = text.lower()
+        character = character.lower()
+        if len(text) >= 150:
+            embed = Embed(
+                title=random.choice(NEGATIVE_REPLIES),
+                description="The given text is too long! Please submit something under 150 characters.",
+                color=Colours.soft_red
+            )
+            await ctx.send(embed=embed)
+            return
+
+        try:
+            msgbody = get_output_string(character, text)
+        except Exception:
+            embed = Embed(
+                title=random.choice(NEGATIVE_REPLIES),
+                description="That is an invalid character! Please enter a valid one.",
+                color=Colours.soft_red
+            )
+            await ctx.send(embed=embed)
+            return
+        if character in ["dragon", "trex", "stegosaurus"]:
+            embed = Embed(
+                title=random.choice(NEGATIVE_REPLIES),
+                description="The given character is invalid! Please submit a valid character.",
+                color=Colours.soft_red
+            )
+            await ctx.send(embed=embed)
+            return
```

Comment:
`dragon`, `stegosaurus` and `trex` are valid characters and are part of the library.

 Can you include the reason why they are not being allowed in the command?

Reply:
Also, this should be done before we format the message for the cowsay output. i.e. line 44

Reply:
Breaks message length

Reply:
> Breaks message length

That should be mentioned in the code then as a comment,